common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -198,11 +198,13 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
           val linkToInteractive = link.first().attr("href")
           val iframe = document.createElement("amp-iframe")
           val overflowElem = document.createElement("div")
+          val placeholderElem = document.createElement("amp-img")
+
           // In AMP, when using the layout `responsive`, width is 100%,
           // and height is decided by the ratio between width and height.
           // https://www.ampproject.org/docs/guides/responsive/control_layout.html
           iframe.attr("width", "5")
-          iframe.attr("height", "1")
+          iframe.attr("height", "3")
```

Comment:
@NataliaLKB Hard to find a size that works for all really, 5:3 seems like a reasonable guess!